docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -0,0 +1,338 @@
+    "- Alternative to the dual optimization from the `scikit-learn` package.\n",
+    "- Finds a $\\delta$ accurate solution.\n",
+    "\n",
+    "This algorithm yields a training complexity that is independent of the size of the training set, such that the `PegasosQSVC` is expected to train faster than QSVC for sufficiently large training sets:\n",
+    "\n",
+    "|     | Dual optimization | Pegasos algorithm|\n",
+    "| --- | --- | --- |\n",
+    "| Ideal kernel evaluations (classical) | $\\mathcal{O}(M^2)$ | $\\mathcal{O}(\\frac{M}{\\delta})$ |\n",
+    "| Ideal quantum circuit evaluations | $\\mathcal{O}(\\frac{M^{4.67}}{\\epsilon^2})$ | $\\mathcal{O}(\\text{min}[\\frac{M^2}{\\delta^3}, \\frac{1}{\\delta^5}])$ |\n",
+    "\n",
+    "- $M$ size of the training set\n",
+    "- $\\epsilon$ absolute difference between ideal and noisy decision function\n",
+    "- $\\delta$ accuracy of the optimization\n",
+    "\n",
+    "The algorithm can be used as direct replacement of `QSVC` with some hyper-parameterization."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "thirty-painting",
+   "metadata": {},
+   "source": [
+    "Let's generate some data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "impressed-laser",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import make_blobs\n",
+    "\n",
+    "# example dataset\n",
+    "features, labels = make_blobs(n_samples=20, n_features=2, centers=2, random_state=3, shuffle=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "moderate-yugoslavia",
+   "metadata": {},
+   "source": [
+    "We pre-process the data to ensure compatibility with the rotation encoding and split it into the training and test datasets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "adolescent-composer",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.preprocessing import MinMaxScaler\n",
+    "\n",
+    "features = MinMaxScaler(feature_range=(0, np.pi)).fit_transform(features)\n",
+    "\n",
+    "train_features, test_features, train_labels, test_labels = train_test_split(features, labels, \n",
+    "                                                                            train_size=15, shuffle=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "central-poverty",
+   "metadata": {},
+   "source": [
+    "We have two features in the dataset, so we set a number of qubits to the number of features in the dataset.\n",
+    "\n",
+    "Then we set $\\tau$ to the number of steps performed during the training procedure. Please note that, there is no early stopping criterion in the algorithm. The algorithm iterates over all $\\tau$ steps.\n",
+    "\n",
+    "And the last one is the hyperparameter $C$, the larger $C$ is, the faster the algorithm runs."
```

Comment:
This needs more explanation, otherwise, why wouldn't you always set `C` to infinity?

Reply:
This is a good question, I noticed `C` is large compared to classical SVM, but did not have chance to review the algorithm in details. This is taken as is from the @Arne-Thomsen 's notebook.
@Arne-Thomsen Can you clarify a bit here?

Reply:
OK, I replaced this sentence with the docstring. Now it is: `And the last one is the hyperparameter 𝐶. Positive regularization parameter. The strength of the regularization is inversely proportional to 𝐶. Must be strictly positive. The penalty is a L1 penalty.`